### DISCUSSION TITLE
🎉 New Release: What's New and Help windows; File menu access (v1.13)

### DISCUSSION THREAD

**Discussion by chrisgurney** [chrisgurney/obsidian-note-toolbar#143] [Announcements]
See the [release notes for v1.13](https://github.com/chrisgurney/obsidian-note-toolbar/releases/tag/1.13), now available in Community Plugins.

Thank you to these contributors for their ideas, and issues: @imeed166 @l-t-k 

In case you missed it, [check out v1.12](https://github.com/chrisgurney/obsidian-note-toolbar/releases/tag/1.12): Quick access to tools; DE, UK, ZH translations

**Comment by imed-ghomari** [comment]
Hi, thank you for the quick release. I have an issue though, when I click on the toolbar on the right click menu, the current file is the one affected, not the linked file.

**Reply by chrisgurney** [reply]
@imeed166 Ahhhhhhh yes, I guess that makes sense as that's what you wanted to do. I think I was focussed on the wrong thing here.

Can you provide an example (script and such) that I can run locally to see if it's possible? Something simple would be fine. Otherwise I'm not sure this is even helpful for anybody.

**Reply by imed-ghomari** [reply]
Sure, here is a templater script that would change a date type property named "review" to today :

```
<%* 
const date = moment().format("YYYY-MM-DD") 
const file = tp.file.find_tfile(tp.file.title);
await app.fileManager.processFrontMatter(file, (frontmatter) => {
  frontmatter["review"] = date;
});
_%>
```
Link it to a command in the templater plugin setting, and then link the command to a toolbar item. I'm sure you know the rest xp.

**Reply by chrisgurney** [reply]
@imeed166 Should be fixed in [1.13.3](https://github.com/chrisgurney/obsidian-note-toolbar/releases/tag/1.13.3). 

The way it works is to temporarily open the file in a sidebar, execute the command, then close the sidebar. You will see a flicker. 

I haven't tested this with longer commands/scripts, so I'm not sure how that might work. Feedback would be appreciated.

It worked with the script you provided.
